pkg/flink/internal/controller/input_controller.go

```diff
@@ -218,17 +212,35 @@ func getUnixBindings() []prompt.Option {
 	return []prompt.Option{
 		prompt.OptionAddASCIICodeBind(
 			prompt.ASCIICodeBind{
-				ASCIICode: []byte{0x1b, 0x62}, // Alt/Option + Arrow Left
+				ASCIICode: []byte{0x1b, 0x62}, // Alt/Option + Arrow Left (ESC + b)
```

Comment:
```suggestion
				ASCIICode: []byte{0x1b, 0x62}, // Alt/Option + Arrow Left (legacy ESC + b)
```
Do you think adding this makes sense? And then updating all comments to follow the same format